src/components/data_comps/docn/cime_config/namelist_definition_docn.xml

```diff
@@ -254,7 +256,7 @@
     <type>char</type>
     <category>streams</category>
     <group>shr_strdata_nml</group>
-    <valid_values>SSTDATA,SST_AQUAP1,SST_AQUAP2,SST_AQUAP3,SST_AQUAP4,SST_AQUAP5,SST_AQUAP6,SST_AQUAP7,SST_AQUAP8,SST_AQUAP9,SST_AQUAP10,SOM,SOM_AQUAP,IAF,NULL,COPYALL</valid_values>
+    <valid_values>SSTDATA,SST_AQUAP1,SST_AQUAP2,SST_AQUAP3,SST_AQUAP4,SST_AQUAP5,SST_AQUAP6,SST_AQUAP7,SST_AQUAP8,SST_AQUAP9,SST_AQUAP10,SST_AQUAPFILE,SOM,SOM_AQUAP,IAF,NULL,COPYALL</valid_values>
```

Comment:
The datamode in namelist_definition.xml and the DOCN_MODE are not the same thing. The DOCN_MODE maps to a datamode that is used directly by the docn model. For now DOCN_MODE copyall is not used - so I removed it. But the DOCN datamodel value of COPYALL could still be used at some point - so it should stay in the valid values.